scripts/integration/docker-compose.datadog-traces.yml

```diff
@@ -2,7 +2,9 @@ version: "3"
 
 services:
   datadog-trace-agent:
-    image: docker.io/datadog/agent:7.39.1
+    # "latest" tag is always the most recent stable release
+    # this is a fallback in case "breaking" changes in the Agent aren't surfaced to Vector
+    image: docker.io/datadog/agent:latest
```

Comment:
I thought our tests _had_ broken because we were tracking latest, am I confused?

Reply:
Yesterday our `datadog_agent` int test was broken because we needed to adapt it to the changes that were released in the latest Agent. The fix required explicitly setting `DD_HOSTNAME`.

The dd traces int test was not failing, because it was not set to track latest yet.
I was planning to do that as part of these changes, so I added that change + the `DD_HOSTNAME` setting in this file as well.